src/osx/carbon/font.cpp

```diff
@@ -295,6 +295,11 @@ wxFontRefData::wxFontRefData(wxOSXSystemFont font, int size)
 
 static const CGAffineTransform kSlantTransform = CGAffineTransformMake( 1, 0, tan(wxDegToRad(11)), 1, 0, 0 );
 
+typedef struct {
```

Comment:
This is C-ism, there is really no need for `typedef` in C++, just `struct OSXFontEntry` would do.

Also, not strictly related, but I'd take this inside an anonymous namespace (`namespace {...}`) to prevent this class name from leaking into the global namespace.